### ISSUE TITLE
xfreeRDP should throw an error in case the drive syntax is incorrect

### ISSUE THREAD

**Issue by axos88** [FreeRDP/FreeRDP#3844]
When supplying the parameters to /drive incorrectly, like so:
```
/opt/freerdp-nightly/bin/xfreerdp  /drive:sharedtmp:/tmp /u:**** /v:**** /p:****
```

It is silently ignored, rather than throwing an error in my face, and suggesting to use the correct syntax:

```
/opt/freerdp-nightly/bin/xfreerdp  /drive:sharedtmp,/tmp /u:**** /v:**** /p:****
```

**Comment by akallabeth** [comment]
Basically #3574. Needs some work, pull requests are welcome ;)

**Comment by akallabeth** [comment]
Fix in #4297

**Comment by akallabeth** [comment]
Works for me with current master. Reopen if still an issue.
